EventKit/eventkit-cloud
# Product list response

# Description of Improvement

Sorting of providers by "Most Downloaded" and "Recently Downloaded" have been altered to respond faster.
Sorting is no longer done by ranks, but counts.
"Most Downloaded" sorts from highest download count to lowest.
"Recently Downloaded" has been changed to translate date into number of weeks since last downloaded. (0 being this week)

## How to test

Either test with the `/eventkit_cloud/ui/static/ui/app/tests/CreateDataPack/ExportInfo.spec.tsx` file, or modify some providers to see the change. 
> For example: Adding `latest_download: 3,` and `download_count: 1,` means that it was downloaded once, ~3 weeks ago

## Quality Assurance

The following items have been updated:


## Screenshots

NA
